RecoLocalCalo/HGCalRecProducers/test/testHGCalRecoLocal_cfg.py

```diff
@@ -1,7 +1,7 @@
 import FWCore.ParameterSet.Config as cms
 from Configuration.StandardSequences.Eras import eras
 
-process = cms.Process("testHGCalRecoLocal",eras.Phase2C2_timing)
+process = cms.Process("testHGCalRecoLocal",eras.Phase2C2)
```

Comment:
@lgray 
is this change OK?

@bsunanda prefers this based on https://github.com/cms-sw/cmssw/pull/16524#issuecomment-260507586